webapp/src/components/sidebar/sidebarUserMenu.tsx

```diff
@@ -20,32 +20,32 @@ import RegistrationLink from './registrationLink'
 
 import './sidebarUserMenu.scss'
 
-type Props = {
-    whiteLogo: boolean
-    showVersionBadge: boolean
-    showAccountActions: boolean
-}
+type Props = {}
 
 const SidebarUserMenu = React.memo((props: Props) => {
     const history = useHistory()
     const [showRegistrationLinkDialog, setShowRegistrationLinkDialog] = useState(false)
-    const {whiteLogo, showVersionBadge, showAccountActions} = props
+    const showVersionBadge = (window as any).isFocalboardPlugin === true
+    const focalboardTitle = (window as any).isFocalboardPlugin !== true
+    const showAccountActions = (window as any).isFocalboardPlugin !== true
```

Comment:
Same for these, if we want to cover for the `null` cases we could either create new bools or use the `!!` trick for the first one and just negate the other two

Reply:
I think I going to move all this to a utility function that returns a boolean.